clang/test/CodeGenSYCL/intel-fpga-local.cpp

```diff
@@ -18,12 +18,15 @@
 //CHECK: [[ANN16:@.str.[0-9]*]] = {{.*}}foobar
 //CHECK: [[ANN17:@.str.[0-9]*]] = {{.*}}{memory:MLAB}{sizeinfo:4,500}
 //CHECK: [[ANN18:@.str.[0-9]*]] = {{.*}}{memory:BLOCK_RAM}{sizeinfo:4,10,2}
+//CHECK: [[ANN19:@.str.[0-9]*]] = {{.*}}{memory:DEFAULT}{sizeinfo:4}{numbanks:4}{bank_bits:4,5}
```

Comment:
Github is not allowing me to add comments to not modified strings for some reasons. Could you please add a compilation string for sycl-host? Probably I missed it one of previous reviews, sorry for that. I want to ensure, that in the correct cases we emit no diagnostics during host compilation. It can be done out of this change scope in a separate patch.

Reply:
Agree, it's needed to be improved. I will add this case in a separate PR